packages/openapi-types/index.ts

```diff
@@ -234,6 +243,7 @@ export namespace OpenAPIV3_1 {
     }
   >;
 
+  // TODO comment
   export type CallbackObject = Record<string, PathItemObject | ReferenceObject>;
```

Comment:
This is extensible according to the specification, but a limitation of TypeScript means we can't add the `x-${string}` signature as it would be shadowed by the `string` one (which we can't constrain any further).